src/components/PreviewArticle.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react"
+import {params, useParams} from "react-router-dom"
+import {getArticleById} from "../api"
+import LikeArticle from "./LikeArticle"
+import "./previewArticle.css"
+function PreviewArticle() {
+    const [article, setArticle] = useState({})
+    const {article_id} = useParams()
+    useEffect(() => {
+        getArticleById(article_id).then((article) => {
+            setArticle(article)
+        })
+    }, [article_id])
+    return (
```

Comment:
You should have a loading state here for when the review has not yet been fetched, providing the user with some visual indication that the page is still loading.